ScoutSuite/core/processingengine.py

```diff
@@ -40,38 +37,30 @@ def run(self, cloud_provider, skip_dashboard=False):
                 path = finding_path.split('.')
                 service = path[0]
                 manage_dictionary(cloud_provider.services[service], self.ruleset.rule_type, {})
-                cloud_provider.services[service][self.ruleset.rule_type][rule.key] = {}
-                cloud_provider.services[service][self.ruleset.rule_type][rule.key]['description'] = rule.description
-                cloud_provider.services[service][self.ruleset.rule_type][rule.key]['path'] = rule.path
+
+                rule_dict = {"description": rule.description, "path": rule.path}
                 for attr in ['level', 'id_suffix', 'class_suffix', 'display_path']:
                     if hasattr(rule, attr):
-                        cloud_provider.services[service][self.ruleset.rule_type][rule.key][attr] = getattr(rule, attr)
+                        rule_dict[attr] = getattr(rule, attr)
                 try:
                     setattr(rule, 'checked_items', 0)
-                    cloud_provider.services[service][self.ruleset.rule_type][rule.key]['items'] = recurse(
-                        cloud_provider.services, cloud_provider.services, path, [], rule, True)
+                    for a in dir(rule):
```

Comment:
Wouldn't this loop include the attributes set in the previous for loop? (lines 42-44)

Reply:
man, now you're testing me.  lol.  

I believe I removed the lines in 43-45 (https://github.com/nccgroup/ScoutSuite/pull/791/files#diff-3bd3294a5da787ffbbd439e6ac8f6a1aL43-L45) as the looping in this group was inclusive of those, so it didn't need to be perforemd.